apps/web/app/api/cron/weekly-summary/components/LiveSurveyNotification.tsx

```diff
@@ -0,0 +1,106 @@
+import { TNotificationDataSurvey, TSurveyResponse } from "@/app/api/cron/weekly-summary/types";
+import { Container, Hr, Link, Tailwind, Text } from "@react-email/components";
+
+import { EmailButton } from "@formbricks/lib/emails/EmailButton";
+
+const getButtonLabel = (count: number): string => {
+  if (count === 1) {
+    return "View Response";
+  }
+  return `View ${count > 2 ? count - 1 : "1"} more Response${count > 2 ? "s" : ""}`;
+};
+
+const convertSurveyStatus = (status: string): string => {
+  const statusMap = {
+    inProgress: "Live",
+    paused: "Paused",
+    completed: "Completed",
+  };
+
+  return statusMap[status] || status;
+};
+
+interface LiveSurveyNotificationProps {
+  WEBAPP_URL: string;
+  environmentId: string;
+  surveys: TNotificationDataSurvey[];
+}
+
+export const LiveSurveyNotification = ({
+  WEBAPP_URL,
+  environmentId,
+  surveys,
+}: LiveSurveyNotificationProps) => {
+  const createSurveyFields = (surveyResponses: TSurveyResponse[]) => {
+    if (surveyResponses.length === 0) {
+      return (
+        <Container className="mt-4">
+          <Text className="m-0 font-bold">No Responses yet!</Text>
+        </Container>
+      );
+    }
+    let surveyFields: JSX.Element[] = [];
+    const responseCount = surveyResponses.length;
+
+    surveyResponses.forEach((response, index) => {
+      if (!response) {
+        return;
+      }
+
+      for (const [headline, answer] of Object.entries(response)) {
+        surveyFields.push(
+          <Container className="mt-4" key={`${index}-${headline}`}>
+            <Text className="m-0">{headline}</Text>
+            <Text className="m-0 font-bold">{answer.toString()}</Text>
+          </Container>
```

Comment:
![image](https://github.com/formbricks/formbricks/assets/56182734/21d902e4-1600-4f93-89c3-6e86eee588c7)
can we transform the response of the picture selection question to display the option image instead of the id?
Also, in the case of upload-type questions, can we show a grey box like `Response Finished Email`?
And in matrix type questions, it is [object object], can you please check the parsing issue?